homeassistant/components/anthropic/const.py

```diff
@@ -13,3 +13,8 @@
 RECOMMENDED_MAX_TOKENS = 1024
 CONF_TEMPERATURE = "temperature"
 RECOMMENDED_TEMPERATURE = 1.0
+CONF_THINKING_BUDGET_TOKENS = "thinking_budget_tokens"
+RECOMMENDED_THINKING_BUDGET_TOKENS = 0
```

Comment:
Shouldn't we just set it to 1024. Now a user would change the model and then has to figure out what the minimum is.

Reply:
We say that in the description text from strings.json, but I can change it if you want

Reply:
Done